src/app/admin-view/components/search/search.component.ts

```diff
@@ -0,0 +1,31 @@
+import {Component, EventEmitter, OnInit, Output} from '@angular/core';
+import {FormControl, FormGroup} from '@angular/forms';
+
+@Component({
+  selector: 'app-search',
+  templateUrl: './search.component.html'
+})
+export class SearchComponent implements OnInit {
+
+
+  form = new FormGroup({
```

Comment:
toto sa neda presunut do `ngOnInit` ?

Reply:
Ked som sa dival na oficialnu dokumentaciu pouzivali to takto. Pretoze ts to automaticky zahrnie o konstruktora daneho komponentu takze ten objekt bude dostupny este pre tym nez sa inicializuju inputy a outputy co je bod ked sa spusta ngOnInit.